.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
 #### Checklist
-- [ ] Unit tests
-- [ ] Updated testplan
+- [ ] Functionality meets specifications
+- [ ] Functionality working
```

Comment:
There can be bugs in a part of the functionality that wasn't detailed in the specifications. Alternatively, you can have working functionality that doesn't meet the specifications.